### DISCUSSION TITLE
Constructing Parameter/Configuration (.yaml) file for ultrasound images

### DISCUSSION THREAD

**Discussion by fixture94** [jries/SMAP#24] [General]
Based off of the work out of Dr. Song's lab at the University of Illinois, we are trying to apply DECODE methods to superresolve microbubble images, but I'm running into a roadblock as I am trying to create the relevent parameter file. Does anyone have any advice or guidance on creating that relevent file? 

Thank you,

**Comment by Haydnspass** [comment]
Hey,
so if you start with SMAP and have done localization, you can create the relevant training parameter file with the plugin 
<img width="582" alt="image" src="https://github.com/user-attachments/assets/fc6f298b-8c42-455a-949d-d2ab4f34890e">

The most relevant parameters are background and the photon distribution. It is also okay to thumb-guess and refine them. Rather estimate to broad than too narrow.

Hope this helps,

Best,
Lucas
